packages/admin-stories/src/docs/components/SplitButton/SplitButton.stories.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Components/SplitButton" />
+
+## Uncontrolled Split Button
+
+<Canvas>
+    <Story id="stories-components-split-button-uncontrolled--uncontrolled" />
+</Canvas>
+
+## Controlled Split Button
+
+<Canvas>
+    <Story id="stories-components-split-button-controlled--controlled" />
+</Canvas>
+
+## LocalStorage Index
+
+<Canvas>
+    <Story id="stories-components-split-button-persistent-index--local-storage-index" />
+</Canvas>
+
+## SessionStorage Index
+
+<Canvas>
+    <Story id="stories-components-split-button-persistent-index--session-storage-index" />
+</Canvas>
+
+## One Button
+
+with only one Child the `SplitButton` automatically hides Split Button Selector, but it can be enabled by manually setting the `showSelectButton={true}`s
```

Comment:
```suggestion
With only one Child the `SplitButton` automatically hides Split Button Selector, but it can be enabled by manually setting the `showSelectButton={true}`s
```